compiler/rustc_hir_typeck/src/op.rs

```diff
@@ -335,7 +335,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                                 format!("cannot divide `{lhs_ty}` by `{rhs_ty}`")
                             }
                             hir::BinOpKind::Rem => {
-                                format!("cannot mod `{lhs_ty}` by `{rhs_ty}`")
+                                format!("cannot rem `{lhs_ty}` by `{rhs_ty}`")
```

Comment:
Actually, let's expand the wording here to make it even clearer (feel free to change the wording further), here and elsewhere

```suggestion
                                format!("cannot calculate the remainder of `{lhs_ty}` divided by `{rhs_ty}`")
```

Reply:
Good idea, fixed it.